packages/queued-request-controller/tests/QueuedRequestController.test.ts

```diff
@@ -0,0 +1,211 @@
+import type { QueuedRequestControllerOptions } from '../src';
```

Comment:
At some point in the (recent) past, I had put tests 'colocated', and you asked me to put them in a folder named test. (see here https://github.com/MetaMask/core/pull/1087#discussion_r1093762079)
https://github.com/MetaMask/core/pull/1087#discussion_r1093762079

Reply:
Ah, I think I might have miscommunicated. At the time I meant only `helpers.ts`, `shared-tests.ts`, and files like `no-block-param.ts`, but not `NetworkController.test.ts` itself. It was a bit confusing in that case because `no-block-param.ts` and `shared-tests.ts` also contain tests.

I guess a better rule here would be: if it ends in `.test.ts`, place it in `src/` and not `tests/` (so it sits next to the file it tests). Does that make sense?

Reply:
Gotcha. In that case, I've moved them.